Cilsil/Services/CfgParserService.cs

```diff
@@ -199,6 +199,14 @@ private void ComputeMethodCfg(MethodDefinition method)
                     programState.PushInstruction(methodBody.Instructions.First(), initNode);
                     do
                     {
+                        if (programState.IsTopSnapshotInstructionNull())
```

Comment:
Considering at line 199 an instruction was pushed into `InstructionStack` and the while loop is condition on `InstructionStack.count > 0`, why is this case possible?

Reply:
I haven't looked too deeply into the root causes, but this is a very rare case that was calling a nullreference exception in the translation, so I handle it here (and log it)